docs/adr/adr-009-telemetry.md

```diff
@@ -0,0 +1,226 @@
+# ADR #009: Telemetry
+
+## Changelog
+
+* 2022-07-4: Started
+* 2022-07-10: Initial Draft finished
+* 2022-07-11: Stylistic improvements
+
+## Authors 
+
+@Wondertan @liamsi
+
+## Legend
+- "ShrEx" - Share Exchange
+> It's all ogre now
+
+## Context
+> Now I know why I don't like writing ADRs - because I cannot run/test them and see if they work or not.
+> Hoping that quality team feedback will solve this problem!
+
+Celestia Node needs deeper observability of each module and their components. The only integrated observability solution 
+we have is logging and there are two more options we need to explore from the observability triangle(tracing, metrics and logs).
+
+There are several priorities and "why"s we need deeper observability:
+* Analysis of the current p2p share exchange or "ShrEx" stack
+  * So we can evaluate real world Full Node reconstruction qualities, along with data availability sampling
+  * And adjust our roadmap accordingly.
+* Incentivized Testnet
+  * Tracking participants and validation that do task correctly
```

Comment:
```suggestion
  * Tracking participation and progress on incentivized testnet tasks.
```
validators are tracked differently.

Reply:
@liamsi, This is not about _validators_, but about _validation_ of participants

Reply:
As we discussed with @YazzyYaz, there is going to be a different set of tasks for which participants will get additional points. They may and may not take this. We need to _verify_ what they claimed to do.